### PR TITLE
React: add notification badge

### LINKED ISSUE #112: Add notification badges

### PR DESCRIPTION
**Pull request by annnag**
Closes #112

### CODE REVIEW

File: react/src/pages/NotificationPopover.tsx [hunk 1]
```diff
@@ -66,7 +68,9 @@ export default function NotificationPopover({ lastLoginTime }: NotificationPopov
         <div>
             <IconButton onClick={handlePopoverOpen}>
                 <Tooltip title="See notifications" arrow>
-                    <NotificationsActive fontSize='large' style={{fill: "white"}} />
+                    <Badge badgeContent={numNotification} color="secondary" showZero>
```
Comment (frewmack) [inline react/src/pages/NotificationPopover.tsx:71]: Looks good! My only suggestion is that the badge should not show if there are no notifications.
```suggestion
                    <Badge badgeContent={numNotification} color="secondary">
```

**Review by frewmack — CHANGES_REQUESTED** [review]

**Review by frewmack — APPROVED** [review]
Looks good to me

File: react/src/pages/NotificationPopover.tsx [hunk 2]
```diff
@@ -66,7 +68,9 @@ export default function NotificationPopover({ lastLoginTime }: NotificationPopov
         <div>
             <IconButton onClick={handlePopoverOpen}>
                 <Tooltip title="See notifications" arrow>
-                    <NotificationsActive fontSize='large' style={{fill: "white"}} />
+                    <Badge badgeContent={numNotification} color="secondary">
```
Comment (kevinlul) [inline react/src/pages/NotificationPopover.tsx:71]: You don't need another state variable to track this. You can just use `analyses.length` here.

**Review by kevinlul — CHANGES_REQUESTED** [review]
A preview for everybody

Screen Shot 2020-10-29 at 11 15 06

**Review by kevinlul — APPROVED** [review]
LGTM

**Pull request merged**